code/modules/virus2/antibodies.dm

```diff
@@ -1,4 +1,3 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:33
```

Comment:
What a monster

Reply:
Most of this file will go in the next part.